iOSClient/Settings/NCKeychain.swift

```diff
@@ -609,6 +609,18 @@ import KeychainAccess
         return (data, password)
     }
 
+    // MARK: - Albums
+
+    func setAutoUploadAlbumIds(account: String, albumIds: [String]) {
+        let key = "AlbumIds" + account
+        keychain[key] = albumIds.joined(separator: ",")
+    }
+
+    func getAutoUploadAlbumIds(account: String) -> [String]? {
+        let key = "AlbumIds" + account
+        return try? keychain.get(key)?.components(separatedBy: ",")
+    }
```

Comment:
Why the optional? Replace with an empty array returned in case of error instead, then you do not have to write the nil-coalescing (`?? []`) redundantly at the call points. The unused additional semantics of "something went wrong" in case of a `nil` return value are not used.
```suggestion
    func getAutoUploadAlbumIds(account: String) -> [String] {
        let key = "AlbumIds" + account
        return try? keychain.get(key)?.components(separatedBy: ",") ?? []
    }
```